<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>geometriesDatabaseTab</class>
 <widget class="QDialog" name="geometriesDatabaseTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometries Database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="vMainLayout" stretch="1,6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,3,0,2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
         <item>
          <widget class="QPushButton" name="newDatabase">
           <property name="toolTip">
            <string>Create new database</string>
           </property>
           <property name="text">
            <string>New</string>
           </property>
           <property name="icon">
            <iconset resource="../../MUSEN/QT_GUI/qt_gui.qrc">
             <normaloff>:/QT_GUI/newPage.png</normaloff>:/QT_GUI/newPage.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadDatabase">
           <property name="text">
            <string>Load</string>
           </property>
           <property name="icon">
            <iconset resource="../../MUSEN/QT_GUI/qt_gui.qrc">
             <normaloff>:/QT_GUI/folderr.png</normaloff>:/QT_GUI/folderr.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveDatabase">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="../../MUSEN/QT_GUI/qt_gui.qrc">
             <normaloff>:/QT_GUI/save_button.png</normaloff>:/QT_GUI/save_button.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveDatabaseAs">
           <property name="text">
            <string>Save as</string>
           </property>
           <property name="icon">
            <iconset resource="../../MUSEN/QT_GUI/qt_gui.qrc">
             <normaloff>:/QT_GUI/Pictures/save_as.png</normaloff>:/QT_GUI/Pictures/save_as.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="geometriesList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="upGeometry">
           <property name="text">
            <string>Up</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downGeometry">
           <property name="text">
            <string>Down</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addGeometry">
           <property name="toolTip">
            <string>Import geometry from STL file (text format)</string>
           </property>
           <property name="text">
            <string>Import</string>
           </property>
           <property name="icon">
            <iconset resource="../../MUSEN/QT_GUI/qt_gui.qrc">
             <normaloff>:/QT_GUI/Pictures/add.png</normaloff>:/QT_GUI/Pictures/add.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteGeometry">
           <property name="toolTip">
            <string>Remove geometry from database</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="../../MUSEN/QT_GUI/qt_gui.qrc">
             <normaloff>:/QT_GUI/Pictures/minus.png</normaloff>:/QT_GUI/Pictures/minus.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonExport">
           <property name="text">
            <string>Export</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Selected geometry</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="selGeometryTable">
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <row>
             <property name="text">
              <string>Name</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Faces number</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Length (X) [m]  </string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Depth (Y) [m]</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Height (Z) [m]</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>New Column</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1">
       <item>
        <widget class="CGeomOpenGLView" name="geomOpenGLView" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CGeomOpenGLView</class>
   <extends>QWidget</extends>
   <header>GeomOpenGLView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>newDatabase</tabstop>
  <tabstop>loadDatabase</tabstop>
  <tabstop>saveDatabase</tabstop>
  <tabstop>saveDatabaseAs</tabstop>
  <tabstop>geometriesList</tabstop>
  <tabstop>upGeometry</tabstop>
  <tabstop>downGeometry</tabstop>
  <tabstop>addGeometry</tabstop>
  <tabstop>deleteGeometry</tabstop>
  <tabstop>pushButtonExport</tabstop>
  <tabstop>selGeometryTable</tabstop>
 </tabstops>
 <resources>
  <include location="../../MUSEN/QT_GUI/qt_gui.qrc"/>
 </resources>
 <connections/>
</ui>
